src/main/resources/lang/en_US.yml

```diff
@@ -33,6 +33,8 @@ GUI_CONFIG_REMOVE_ORIGINAL: '<green>Removing original Wandering Trader trades'
 GUI_CONFIG_KEEP_ORIGINAL: '<red>Not removing original Wandering Trader trades'
 GUI_CONFIG_REFRESH: '<green>Auto-refreshing trades of /wt summoned traders'
 GUI_CONFIG_NO_REFRESH: '<red>Not auto-refreshing trades of /wt summoned traders'
+GUI_CONFIG_PREVENT_NIGHT_INVISIBILITY: '<green>Preventing use of invisibility potions of /wt summoned traders'
```

Comment:
Please copy the English strings into the other message configs as well, so that they all contain every key. This way it's easier for someone who knows the language to see what needs updating at a glance.